s5_base/run.sh

```diff
@@ -104,29 +101,53 @@ fi
 
 # train a first delta + delta-delta triphone(tri1) system on all utterances
 if [ $stage -le 6 ]; then
+  echo "Train deltas triphone1 model"
   steps/train_deltas.sh --boost-silence 1.25 --cmd "$train_cmd" \
     2000 10000 data/train data/lang exp/mono_ali_train exp/samromur_tri1
-fi
 
-exit 0
-# TODO: before decoding create a tgmed or tglarge arpalm
-if [ $stage -eq 7 ]; then
   # decode using the tri1 model
+  echo "Create deltas decoding graph"
   (
     utils/mkgraph.sh data/lang_3gsmall \
       exp/samromur_tri1 exp/samromur_tri1/graph_3gsmall
     for test in dev; do
-      steps/decode.sh --nj 5 --cmd "$decode_cmd" exp/samromur_tri1/graph_tgsmall \
+      steps/decode.sh --nj 5 --cmd "$decode_cmd" exp/samromur_tri1/graph_3gsmall \
       data/$test exp/samromur_tri1/decode_3gsmall_$test
-      steps/lmrescore.sh --cmd "$decode_cmd" data/lang_{3gsmall,tgmed} \
-        data/$test exp/samromur_tri1/decode_{3gsmall,tgmed}_$test
+      steps/lmrescore.sh --cmd "$decode_cmd" data/lang_{3gsmall,3gmed} \
+        data/$test exp/samromur_tri1/decode_{3gsmall,3gmed}_$test
```

Comment:
I think we can skip the medium sized model and just rescore with a big one. Or have you seen somewhere a recommendation to do it this way?

Reply:
In the next acoustic model we use large. Do you want to just not use med at all? even for these intermediate acoustic models and alignments?

Reply:
The rescoring is quick so I find it unnecessary. We would maybe be saving a minute and had to create a whole new LM